test/PerformanceTests/PerformanceTests.csproj

```diff
@@ -6,7 +6,7 @@
   <ItemGroup>
     <PackageReference Include="Azure.Messaging.EventHubs" Version="5.9.2" />
     <PackageReference Include="Microsoft.Azure.WebJobs.Extensions" Version="4.0.1" />
-    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.DurableTask" Version="2.12.0" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.DurableTask" Version="2.13.1" />
```

Comment:
Do we need this `PackageReference` at all? This project references,
```
<ProjectReference Include="..\..\src\DurableTask.Netherite.AzureFunctions\DurableTask.Netherite.AzureFunctions.csproj" />
```
And it can pickup `Microsoft.Azure.WebJobs.Extensions.DurableTask` from that.

Reply:
It is a bit strange - could be an accident. But I'd love to move this to a different PR since this one is old and I'm trying to avoid adding to it's diff too much so we can merge promptly.

Reply:
of course!

Reply:
Created a separate PR: 
https://github.com/microsoft/durabletask-netherite/pull/388/files